src/resources/benefitCompany.ts

```diff
@@ -86,11 +86,11 @@ export const BenefitCompanyResource: ResourceIF = {
   },
   reviewAndConfirm: {
     completingPartyStatement: {
-      certifyStatementHeader: 'the Completing Party, have examined the Benefit Company ' +
+      certifyStatementHeader: 'the Completing Party, have examined the Company ' +
         'Articles and the Incorporation Agreement applicable to the company that is to be ' +
         'incorporated by the filing of this Incorporation Application and confirm that:',
       certifyStatements: [
-        'The Benefit Company Articles and the Incorporation Agreement both contain a signature ' +
+        'The Company Articles and the Incorporation Agreement both contain a signature ' +
```

Comment:
Weird. So this resource, which is for Benefit Companies, should not call them benefit companies in the text?

Reply:
Yea, according to the update designs. This could potentially be taken out of resources but not all the bullet points were identical. 
It seems to change so much, i though it would be best to leave it in resources and not apply some bullet points conditionally.

Reply:
OK, good idea.